src/invert/laplacexy/laplacexy.cxx

```diff
@@ -227,21 +226,22 @@ LaplaceXY::LaplaceXY(Mesh *m, Options *opt, const CELL_LOC loc)
     
     // Convergence Parameters. Solution is considered converged if |r_k| < max( rtol * |b| , atol )
     // where r_k = b - Ax_k. The solution is considered diverged if |r_k| > dtol * |b|.
-    BoutReal rtol, atol, dtol;
-    int maxits; ///< Maximum iterations
-    
-    OPTION(opt, rtol, 1e-5);     // Relative tolerance 
-    OPTION(opt, atol, 1e-10);    // Absolute tolerance
-    OPTION(opt, dtol, 1e3);      // Diverged threshold
-    OPTION(opt, maxits, 100000); // Maximum iterations
-    
+
+    BoutReal rtol = (*opt)["rtol"].doc("Relative tolerance").withDefault(1e-5);
```

Comment:
Mostly as a _style_ thing, I would go with `const` when we can, as then it's clear this is the only place that the variable is touched -- everywhere else must only be reading it.

I'd also be tempted by `const auto`, as the type is (usually) clear from the right, although this doesn't work so well with `std::string`, unless you explicitly use `withDefault<std::string>("value")`.

Just suggestions for the future, rather than asking you to change all these places again! :smile: